lib/python/ray/worker.py

```diff
@@ -553,6 +661,20 @@ def _mode(worker=global_worker):
   """
   return worker.mode
 
+def _export_reusable_variable(name, initializer, reinitializer, worker=global_worker):
+  """Export a reusable variable to the workers. This is only called by a driver.
+
+  Args:
+    name (str): The name of the variable to export.
+    initializer (Callable[[], object]): The function for initializing the
+      reusable variable.
+    reinitializer (Callable[[object], object]): The function for reinitializing
+      the reusable variable.
+  """
+  if _mode() not in [ray.SHELL_MODE, ray.SCRIPT_MODE]:
+    raise Exception("_export_reusable_variable can only be called on a driver.")
+  ray.lib.export_reusable_variable(worker.handle, name, pickling.serialize(initializer), pickling.serialize(reinitializer))
```

Comment:
I would check to see if the two functions are the same (at least an `is` test), and if so, only pickle them once. Pickling is very expensive.